docs/reference/commands/other-commands.rst

```diff
@@ -0,0 +1,5 @@
+:ref:`ref_commands_init`
+   Create an initial project filetree.
+
+:ref:`ref_commands_version`
+   Show the application version and exit.
```

Comment:
I don't quite understand why we needed to transclude these – couldn't we have included this content and `lifecycle-commands.rst` directly in `commands.rst`?

Reply:
I do not know enough sphinx best practices to know if this is needed or not. I suggest reworking that for both files if we want in another PR.

Reply:
Sure, let's earmark this.